telegram-client/helpers/recognizeType.ts

```diff
@@ -0,0 +1,27 @@
+import hashtags from '../config/hashtags';
+import { MessageType } from '../prepareMessage/types';
+
+export const recognizeType = (text: string): MessageType | null => {
+  const hashTagRegex = /(^|\s)(#[a-zA-Zа-яА-Я\d-]+)/gm;
+  const hashes = text.match(hashTagRegex);
+  if (hashes === null || hashes.length === 0) {
+    return null;
+  }
+
+  let messageType: MessageType | null = null;
+
+  hashes.map((hash) => {
+    const preparedHash = hash.replace('#', '').trim();
+    const types = Object.keys(hashtags);
+
+    types.map((type) => {
```

Comment:
`map` should be changed to forEach because map is to return something, but here is just a enumeration